lib/config/defaults.js

```diff
@@ -1472,7 +1472,8 @@ const getResolveDefaults = ({ cache, context, targetProperties, mode }) => {
 			// for backward-compat: Custom Dependency
 			unknown: cjsDeps(),
 			// for backward-compat: getResolve without dependencyType
-			undefined: cjsDeps()
+			undefined: cjsDeps(),
+			"css-import": styleDeps()
```

Comment:
What do you think rename it to just `css`, we need to replace it in `dependency` from `css-import` to `css`, I think it has more sense

Reply:
CSS is already a module type. Wont that cause a collision? 
https://github.com/webpack/webpack/blob/main/lib/css/CssModulesPlugin.js#L632

After renaming css-import to css in ExternalModule and CSSModule, the snapshots are failing.

Reply:
Should not because, it is just a dependecy type